.github/workflows/ci.yml

```diff
@@ -75,23 +85,20 @@ jobs:
         TLS_KEY=$TLS/redis.key \
         TLS_CACERT=$TLS/ca.crt \
         REDIS_BINARY=`command -v redis-server` \
-        pytest --ignore=tests/flow --ignore=test_example.py -v
+        poetry run pytest --ignore=tests/flow --ignore=test_example.py -v
 
-    - name: Install RLTest
-      run: |
-        python3 -m pip install .
 
     - name: Flow Test OSS Single Module
       run: |
         cd tests/flow
         make -C modules
-        CONTAINS_MODULES=1 RLTest --env oss -v --clear-logs --module modules/module1.so
+        poetry run RLTest --env oss -v --clear-logs --module modules/module1.so --module-args "DUPLICATE_POLICY BLOCK"
 
     - name: Flow Test OSS Multiple Modules  --use-slaves
       run: |
         cd tests/flow
         make -C modules
-        CONTAINS_MODULES=1 RLTest --env oss -v --clear-logs \
+        poetry run RLTest --env oss -v --clear-logs \
```

Comment:
Same working-directory comment.

But beyond that, maybe we want to run all tests regardless of pass/fail (if: success() || failure()) and father the failure at the end?

At least this way we get all the outputs of rltest test runs. WDYT?